crates/net/nat/src/lib.rs

```diff
@@ -22,7 +22,7 @@ use tracing::warn;
 
 /// All builtin resolvers.
 #[derive(Debug, Clone, Copy, Eq, PartialEq, Default, Hash)]
-pub enum NatResolver {
+pub(crate) enum NatResolver {
```

Comment:
why making this `pub crate`?

Reply:
I couldn't access the NatResolver from node/mod.rs, didn't wan to redefine the enum.